tests/ios/unit-test/test-imports-app/empty.swift

```diff
@@ -1,9 +1,12 @@
 import Foundation
 import SomeFramework
+import Basic
```

Comment:
this still work with bazel 7 probably because swift compile does not depend on CcInfo